build.gradle

```diff
@@ -1,39 +1,50 @@
 import com.github.spotbugs.SpotBugsTask
-import org.apache.tools.ant.filters.ReplaceTokens
 
 buildscript {
     ext {
-        springBootVersion = '2.1.0.RELEASE'
+        springVersion = '5.2.3.RELEASE'
+        springBootVersion = '2.2.4.RELEASE'
```

Comment:
we shouldn't upgrade the version of spring boot or Gradle as part of this pull request, we need to at least keep some sort of consistency across services with the versions we use

Reply:
Ill add a ticket for sprint 31 to upgrade the other projects. Lets keep this project as is and see if gradle or springboot gives us any obvious problems.